sw/device/sca/aes_serial.c

```diff
@@ -87,9 +89,45 @@ uint8_t batch_plaintexts[kNumBatchOpsMax][kAesTextLength];
 bool sample_fixed = true;
 
 /**
- * Fixed key for fvsr key TVLA batch capture.
+ * The generation key for TVLA batch capture - specified in [DTR] Section 5.1.
+ * This key is used for generating all pseudo-random data for batch captures.
  */
-uint8_t key_fixed[kAesKeyLength];
+const uint8_t key_gen[kAesKeyLength] = {0x12, 0x34, 0x56, 0x78, 0x9a, 0xbc,
+                                        0xde, 0xf1, 0x23, 0x45, 0x67, 0x89,
+                                        0xab, 0xcd, 0xe0, 0xf0};
+/**
+ * An array to store pre-computed round keys derived from key_gen.
+ */
+uint32_t key_gen_round_keys[(kAesKeyLength / 4) * 11];
```

Comment:
Is it worth explicitly zero initializing this? It would be nice if there wasn't undefined behavior in a case someone accidentally sends a `b` or `g` command before initializing (with a `d` command).

That said I don't know this part of the code base well, so maybe this isn't a concern.

Reply:
Fair point @HU90m . I think there are no specific concerns regarding this but it also wouldn't hurt to zero initialize this. I am in favor of initializing.

Reply:
I agree it's a good idea to initialize this array, but rather than zeros I would initialize it to the round-key values derived from key_gen. This way we can also get rid of computing key schedule.
I've pushed these changes.